projects/angular-d3-cloud/src/lib/angular-d3-cloud.component.ts

```diff
@@ -138,7 +169,7 @@ export class AngularD3CloudComponent implements OnChanges, OnInit {
       throw new TypeError(`${AngularD3CloudComponent.TAG}: [fontSizeMapper] must be a function. Current value is: [${this.fontSizeMapper}]`)
     }
 
-    if (!this.fillMapper || typeof this.fillMapper !== 'function') {
+    if (this.fillMapper && typeof this.fillMapper !== 'function') {
```

Comment:
@gmaggiodev what if in runtime the value of fillMapper passed down from parent is null | undefined